### PR TITLE
Changes to a more descriptive name

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/issues/277

### CODE REVIEW

File: sharepoint/sharepoint-server-ps/sharepoint-server/New-SPPowerPointConversionServiceApplication.md [hunk 1]
```diff
@@ -34,10 +34,10 @@ For permissions and the most current information about Windows PowerShell for Sh
 
 ### ------------------EXAMPLE------------------
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-server-ps/sharepoint-server/New-SPPowerPointConversionServiceApplication.md:35]:
```suggestion
### EXAMPLE
```

File: sharepoint/sharepoint-server-ps/sharepoint-server/New-SPPowerPointConversionServiceApplicationProxy.md [hunk 2]
```diff
@@ -30,10 +30,10 @@ The service application proxy is instantiated on the front-end web server and ac
 
 ### ------------------EXAMPLE------------------
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-server-ps/sharepoint-server/New-SPPowerPointConversionServiceApplicationProxy.md:31]:
```suggestion
### EXAMPLE
```

File: sharepoint/sharepoint-server-ps/sharepoint-server/New-SPPowerPointConversionServiceApplicationProxy.md [hunk 3]
```diff
@@ -30,10 +30,10 @@ The service application proxy is instantiated on the front-end web server and ac
...
 ```powershell
-New-SPPowerPointConversionServiceApplicationProxy -Name "MyWorkgroupPPTAppProxy" -ServiceApplication "MyWorkgroupPPTApp" -AddtoDefaultGroup
+New-SPPowerPointConversionServiceApplicationProxy -Name "PowerPoint Conversion Service application proxy" -ServiceApplication "PowerPoint Conversion Service Application" -AddtoDefaultGroup
 ```
 
-This example creates a new instance of the PowerPoint Conversion Service application proxy named MyWorkgroupPPTAppProxy, binds it to the MyWorkgroupPPTApp service application and then adds it to the default service application proxy group
+This example creates a new instance of the PowerPoint Conversion Service application proxy named _PowerPoint Conversion Service application proxy_, binds it to the _PowerPoint Conversion Service Application_ service application and then adds it to the default service application proxy group
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-server-ps/sharepoint-server/New-SPPowerPointConversionServiceApplicationProxy.md:36]:
```suggestion
This example creates a new instance of the PowerPoint Conversion Service application proxy named _PowerPoint Conversion Service application proxy_, binds it to the _PowerPoint Conversion Service Application_ service application, and then adds it to the default service application proxy group
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @tseward

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**